arib/app/auth/auth-http.service.ts

```diff
@@ -23,16 +24,14 @@ export class AuthHttp {
     get(url: string) {
         let headers = new Headers();
         this.createAuthorizationHeader(headers);
-        return this.http.get(this.config.apiURL + "/api/" + url, {
-            headers: headers
-        });
+        return this.http.get(this.config.apiURL + "/api/" + url, { headers })
+            .map((res: Response) => res.json());
```

Comment:
I like returning the actual decoded object, instead of the full Observable<Response>, but what happens now when there is an error? (http 500 for instance?)